### PR TITLE
Added mtaircompressor image

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

File: build/mtaircompressor/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ARG hub
+
+FROM ${hub}/deploy-env:${cycle}
+
+LABEL maintainer Petr Kubánek <pkubanek@lsst.org>
+
+ARG ts_mtaircompressor
+
+RUN source /home/saluser/.setup_sal_env.sh && \
+    conda install -c lsstts ts-mtaircompressor=${ts_adamSensors}
```
Comment (wvreeven) [inline build/mtaircompressor/Dockerfile:11]: `ts_adamSensors` -> `ts_mtaircompressor`

File: build/mtaircompressor/startup.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+source $HOME/.setup_sal_env.sh
+
+run_mtaircompressor.py $RUN_ARG $* &
```
Comment (wvreeven) [inline build/mtaircompressor/startup.sh:5]: I don't think that the `$*` is needed here. If you want to pass on arguments then they need to be added to `RUN_ARG`.

**Review by wvreeven — CHANGES_REQUESTED** [review]
In general this looks good but I found a few errors. Please fix those.

File: build/mtaircompressor/Dockerfile (hunk 1 above)
Comment (pkubanek) [reply build/mtaircompressor/Dockerfile]: Fixed, thanks for pointing that out.

File: build/mtaircompressor/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+WORKDIR /home/saluser/
+
+ENV TS_CONFIG_EAS_DIR=/home/saluser/ts_config_eas
```
[the remark this answers is not in the document]
Comment (pkubanek) [reply build/mtaircompressor/Dockerfile:15]: There isn't any configuration at the moment. Still shall be fine, shouldn't it ?

File: build/mtaircompressor/startup.sh (hunk 2 above)
Comment (pkubanek) [reply build/mtaircompressor/startup.sh]: I need to pass either 1 or 2 (index) when running container. I belive RUN_ARG will be taken only from Dockerfile, not when starting container on Kubernets. But I deleted the $*, assuming somehow magically RUN_ARG would become 1 or 2 when starting different containers.

Comment (wvreeven) [reply build/mtaircompressor/startup.sh]: You're right.

**Review by wvreeven — APPROVED** [review]
Looks good to me.

**Pull request merged**